iml-agent/src/action_plugins/lustre/snapshot.rs

```diff
@@ -18,7 +18,13 @@ pub async fn list(l: List) -> Result<Vec<Snapshot>, ImlAgentError> {
         args.push("--detail");
     }
     let stdout = lctl(args).await?;
-    parse_snapshot_list(&stdout).map_err(ImlAgentError::CombineEasyError)
+    let stdout = stdout.trim();
```

Comment:
Does it cut off new-line as well?

Reply:
Yes: https://play.rust-lang.org/?code=%23!%5Ballow(unused)%5D%0Afn%20main()%20%7B%0Alet%20s%20%3D%20%22%20Hello%5Ctworld%5Ct%22%3B%0A%0Aassert_eq!(%22Hello%5Ctworld%22%2C%20s.trim())%3B%0A%7D&edition=2018